rules/errors.go

```diff
@@ -81,6 +81,7 @@ func NewNoErrorCheck(id string, conf gosec.Config) (gosec.Rule, []ast.Node) {
 	whitelist := gosec.NewCallList()
 	whitelist.AddAll("bytes.Buffer", "Write", "WriteByte", "WriteRune", "WriteString")
 	whitelist.AddAll("fmt", "Print", "Printf", "Println", "Fprint", "Fprintf", "Fprintln")
+	whitelist.AddAll("strings.Builder", "Write", "WriteByte", "WriteRune", "WriteString")
```

Comment:
I think that we have to make this backward compatible with Go 1.9 since these functions were introduced only in Go 1.10. We are trying to keep compatibility with last 3 major versions.

Reply:
How can this be achieved in this case? As this is a string-based whitelist, why is not backward compatible? From my understanding this is not compile code but just strings used for string-matching when comparing AST with the whitelist.